docs/standard/security/fips-compliance.md

```diff
@@ -0,0 +1,29 @@
+---
+title: FIPS compliance - .NET Core
+description: Explains .NET Core Federal Information Processing Standard (FIPS) compliance.
+ms.date: 11/20/2019
+author: "Rick-Anderson"
+ms.author: "riande"
+---
+
+# .NET Core Federal Information Processing Standard (FIPS) compliance
+
+.NET Core:
+
+* Does **not** implement any cryptographic primitives.
+* Passes cryptographic primitives calls through to the standard modules the underlying operating system provides.
+* Does **not** enforce FIPS compliance in .NET Core apps.
+* Does **not** stop developers from using non-FIPS compliant algorithms, even on systems configured for FIPS compliance.
```

Comment:
This seems redundant to the previous line, at least after my change.  Maybe I misunderstand one or the other.